src/platform/packages/private/kbn-ui-shared-deps-npm/moon.yml

```diff
@@ -0,0 +1,44 @@
+$schema: https://moonrepo.dev/schemas/project.json
+id: '@kbn/ui-shared-deps-npm'
```

Comment:
should we create some package linting additional rules that will validate the info of those moon.yml files on CI against what we have in the kibana.jsonc files? (I'm thinking it should be the source of true regarding things already defined there like the owner and we dont want to have deviations in multiple places)

Reply:
until now kibana.jsonc files are the source of truth already . I think we should keep that and linting/fixing/generating where it applies on moon.yml files. (package.json should be excluded from this equation). If you think there are reasons to do it the other way around (moon.yml -> kibana.jsonc) we can discuss it, but we will have to change it across the board.